grails-app/controllers/au/org/ala/merit/HomeController.groovy

```diff
@@ -141,6 +145,15 @@ class HomeController {
     }
 
     def publicHome() {
+        Boolean expSoon = false
```

Comment:
Could we refactor this into its own method and just return the expiryDate or null?  That way we don't need the expSoon flag as well.

Reply:
It also might be better to tie this into the login hook that already exists for recording the user login.  Otherwise we have to make this web service call to ecodata on every single request to the homepage, which is an unnecessary performance hit considering it will be uncommon for this to return an expiry date.